app/src/components/DayOfWeek.js

```diff
@@ -0,0 +1,82 @@
+import * as React from "react";
+
+import Card from "react-bootstrap/Card";
+import ListGroup from "react-bootstrap/ListGroup";
+import ListGroupItem from "react-bootstrap/ListGroupItem";
+
+import * as apiClient from "../apiClient";
+
+const DayOfWeek = ({ date }) => {
+  const [tasks, setTasks] = React.useState([]);
+
+  const loadTasks = async () => setTasks(await apiClient.getTasks());
+
+  React.useEffect(() => {
+    loadTasks();
+  }, []);
+
+  return (
+    <Card>
+      <Card.Img
+        variant="top"
+        src="https://www.icalendar37.net/lunar/api/i.png"
+      />
+      <Card.Body>
+        <Card.Title>{date.format("ddd MMM D YYYY")}</Card.Title>
+        <Card.Text>
+          Some quick example text to build on the card title and make up the
+          bulk of the card's content.
+        </Card.Text>
+      </Card.Body>
+      <ListGroup className="list-group-flush">
+        <ListGroupItem>
+          <AddTask loadTasks={loadTasks} />
+        </ListGroupItem>
+      </ListGroup>
+      <TaskList tasks={tasks} />
+    </Card>
+  );
+};
+
+const TaskList = ({ tasks }) => (
+  <ListGroup className="list-group-flush">
+    {tasks.map(({ id, name }) => (
+      <ListGroupItem key={id}>
+        {name}
+        <button>
+          <i className="bi bi-pencil-square"></i>
+        </button>
+        <button>
+          <i className="bi bi-trash-fill"></i>
+        </button>
+      </ListGroupItem>
+    ))}
+  </ListGroup>
+);
+
+const AddTask = ({ loadTasks }) => {
```

Comment:
I might break these components into their own files - I think of each file as  one 'doing/showing one medium/large feature' component per file.  For the medium/smaller ones it's preference if you do a few in a file or break them up into multiple simple component files 
ie if DayOfWeek has 2 v small components like DayTitle and DayInfo or something, they could all be in the same file.